### DISCUSSION TITLE
Creatable option label "Create XXX" localization

### DISCUSSION THREAD

**Discussion by msmid** [JedWatson/react-select#5266] [Q&A] [answered]
Hi,

I wonder is there a way how to translate `Create "XXX"` in`Creatable` component when user is adding new option?

![image](https://user-images.githubusercontent.com/8785199/178438234-b7a94f05-16da-4eb1-a35c-e5cd69ef894b.png)

**Comment by manjushsh** [comment, marked as answer]
Hi @msmid please refer [this](https://github.com/JedWatson/react-select/issues/2493#issuecomment-416832921) and [docs](https://react-select.netlify.app/props#creatable-props)

[Code Sandbox with basic translation logic](https://codesandbox.io/s/codesandboxer-example-forked-b1g50v?file=/example.tsx)

Pass this to Select Component
```
formatCreateLabel={(userInput) => `[Your translated text here] ${userInput}` }
```

**Reply by msmid** [reply]
Thank you, I missed it in docs
